packages/engine/paima-funnel/package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "assert-never": "^1.2.1",
     "@dcspark/carp-client": "^3.1.0",
-    "@dcspark/cardano-multiplatform-lib-nodejs": "5.2.0"
+    "@dcspark/cardano-multiplatform-lib-nodejs": "5.2.0",
+    "postgres": "^3.3.5"
```

Comment:
To be honest I just didn't think about it. I started with this dep because it was being used in the archive node api, and since I took that as the base it was simpler to get started. Then I was considering porting to pgtyped eventually, but I didn't think about that. Switching to pg should be simple, I'll do that first.